addons/binding/org.openhab.binding.tellstick/README.md

```diff
@@ -8,6 +8,8 @@ All of these 433 MHz devices is one-way, so some versions of the Tellstick monit
 
 The latest versions have also implemented Z-Wave as transmission protocol which open up for more robust transmission due two-ways communication.
 
+Discussion and help for the binding could be found in the community forum thread: https://community.openhab.org/t/tellstick-binding-for-openhab2/
```

Comment:
I would refrain from adding such links here. Discussion and help is available for ALL add-ons in the community, so no need to specifically mention it. And if people use the forum search and type "tellstick", the linked topic is the first hit on the list - so they cannot really miss it.